site_scons/buildutils.py

```diff
@@ -617,92 +704,95 @@ def listify(value):
         # assume `value` is a string
         return value.split()
 
-def removeFile(name):
-    """ Remove file (if it exists) and print a log message """
-    if os.path.exists(name):
-        print('Removing file "%s"' % name)
-        os.remove(name)
 
-def removeDirectory(name):
-    """ Remove directory recursively and print a log message """
-    if os.path.exists(name):
-        print('Removing directory "%s"' % name)
-        shutil.rmtree(name)
+def remove_file(name: Union[Path, str]) -> None:
+    """Remove file (if it exists) and print a log message."""
+    path_name = Path(name)
+    if path_name.exists():
+        logger.info(f"Removing file '{name!s}'")
+        path_name.unlink()
+
+
+def remove_directory(name: Union[Path, str]) -> None:
+    """Remove directory recursively and print a log message."""
+    path_name = Path(name)
+    if path_name.exists() and path_name.is_dir():
+        logger.info(f"Removing directory '{name!s}'")
+        shutil.rmtree(path_name)
+
 
 def ipdb():
     """
     Break execution and drop into an IPython debug shell at the point
     where this function is called.
     """
     from IPython.core.debugger import Pdb
-    from IPython.core import ipapi
+    from IPython.core import getipython
 
-    ip = ipapi.get()
+    ip = getipython.get_ipython()
     def_colors = ip.colors
     Pdb(def_colors).set_trace(sys._getframe().f_back)
 
 
-def getSpawn(env):
+def get_spawn(env):
```

Comment:
PEP 484 ... I believe `env` is specific to `scons`, but there are some ways to define those types.